test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/InheritsWithViewImports_Imports0.cshtml

```diff
@@ -0,0 +1 @@
+﻿@inherits MyPageModel<TModel> foo
```

Comment:
Also, Q: does this ViewImports impact other code generation tests or have we built in a convention to understand `TestName_ImportsX`?